Vagrantfile

```diff
@@ -26,23 +32,21 @@ Vagrant.configure("2") do |config|
     end
   end
 
-  config.vm.define "freebsd" do |c|
-    c.ssh.shell = "csh"
-    c.vm.box = "freebsd/FreeBSD-10.2-RELEASE"
-
+  config.vm.define "freebsd11" do |c|
+    c.vm.box = "freebsd/FreeBSD-11.1-RELEASE"
+    c.vm.guest = :freebsd
+    c.vm.hostname = "freebsd11"
+    c.vm.base_mac = "6658695E16F0"
     c.vm.network "private_network", type: "dhcp"
-    c.vm.synced_folder ".", "/vagrant", type: "nfs"
+    c.vm.synced_folder ".", "/vagrant", disabled: true
```

Comment:
Actually the original version only linked the git repositories. I think that's wrong too as the host still doesn't have access to the files on the client directly. This was likely done to stop `.build` from being shared. Perhaps allowing `$(O)` in the makefile to be modified and setting env automatically to place the VM `.build` outside the shared directory is the best idea.